app/(pages)/_components/Textbox/Textbox.tsx

```diff
@@ -0,0 +1,103 @@
+import styles from './Textbox.module.scss';
+import Image from 'next/image';
+import { useState } from 'react';
+
+interface TextboxProps {
+  name: string;
+  placeholder: string;
+}
+
+const Textbox: React.FC<TextboxProps> = (props) => {
+  const [wordStats, setStats] = useState({
+    words: 0,
+    capacity: false,
+  });
+  function onChange(e: React.ChangeEvent<HTMLTextAreaElement>) {
+    const temp = e.target.value;
+    const tmp = temp.split(/([\S])+/) || [];
```

Comment:
You can use the [:space:] character class to say all whitespace. (this works tho)